### ISSUE TITLE
Archive old feature branches

### ISSUE THREAD

**Issue by scottveirs** [orcasound/orcahello#70]
Get @paulcretu to teach us (at least @scottveirs ) how to archive merged or abandoned feature branches as he has done in the `orcasite` repo, e.g. https://github.com/orcasound/orcasite/tags

**Comment by paulcretu** [comment]
Yep exactly what I've been doing. But only for unmerged/abandoned branches, fully merged branches I just delete.

I actually do `git tag -a archive/<branchname> <branchname> -m ""` because [annotated tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_creating_tags) leave metadata about who tagged it and when. I've been leaving the message blank but could toss in an explanation of why it was archived in there if necessary.

That being said I'm now trying to only do fork-based PRs in orcasound/orcasite. It's a good way to avoid the problem of abandoned branches—although it can still occasionally be useful to absorb an unmerged branch from a fork and archive it so it doesn't get lost.
